coralproject/talk
# [CORL-1549] add go to discussions and go to profile buttons to bottom of stream

## What does this PR do?

Adds "go to discussions" and "go to profile & replies" buttons to bottom of stream if appropriate. Also adds missing localization tags. Unfortunately the `list_alt` icon doesn't look the same as the one in the mockup and on the material icons website, I'm guessing its part of v4.0, which isn't on npm. 
<img width="786" alt="Screen Shot 2020-12-10 at 8 02 36 AM" src="https://user-images.githubusercontent.com/1789029/101797229-ca385080-3ad7-11eb-9b91-ebf4aee01aba.png">
<img width="300" alt="Screen Shot 2020-12-10 at 8 02 22 AM" src="https://user-images.githubusercontent.com/1789029/101797245-cdcbd780-3ad7-11eb-9aa0-cb854205cb31.png">





## What changes to the GraphQL/Database Schema does this PR introduce?
none


## How do I test this PR?

Enable `DISCUSSIONS` feature flag and view bottom of a stream for a logged-in user. click the buttons which should navigate to the respective tabs.
